contributors/devel/e2e-tests.md

```diff
@@ -612,11 +612,8 @@ A quick overview of how we run e2e CI on Kubernetes.
 ### What is CI?
 
 We run a battery of `e2e` tests against `HEAD` of the master branch on a
-continuous basis, and block merges via the [submit
-queue](http://submit-queue.k8s.io/) on a subset of those tests if they fail (the
-subset is defined in the [munger config](https://git.k8s.io/test-infra/mungegithub/mungers/submit-queue.go)
-via the `jenkins-jobs` flag; note we also block on	`kubernetes-build` and
-`kubernetes-test-go` jobs for build and unit and integration tests).
+continuous basis, and block merges via [Tide](https://git.k8s.io/test-infra/prow/cmd/tide)
+on a subset of those tests if they fail.
```

Comment:
This whole section reads like it's left over from the old days of post-submits blocking the submit queue.  In that spirit, I would rewrite this to talk about release-blocking jobs, and reference https://testgrid.k8s.io/sig-release-master-blocking